### ISSUE TITLE
Ui - design.  let's make the review of all improvements

### ISSUE THREAD

**Issue by magsout** [webcompat/webcompat.com#599]
I confess to being a bit lost with all these ideas, and many are still in reflection or perhaps waiting for me to integration. I don't know. I open this issue not discuss the issues, but just to make the point.

- [x] Homepage : https://github.com/webcompat/webcompat.com/issues/571
- [ ] list of issues : https://github.com/webcompat/webcompat.com/issues/545
- [ ] Filters - SearchBar : https://github.com/webcompat/webcompat.com/issues/510
- [ ] hashTag in comment : https://github.com/webcompat/webcompat.com/issues/501
- [ ] Label color : https://github.com/webcompat/webcompat.com/issues/482
- [ ] Activity view : https://github.com/webcompat/webcompat.com/issues/445
- [ ] DashBoard: https://github.com/webcompat/webcompat.com/issues/442
- [x] Responsive list of issues : https://github.com/webcompat/webcompat.com/issues/349

**Comment by karlcow** [comment]
@magsout Good summary.

On some these issues, there are no good solutions yet or there are meta issues. This one you just open becoming a META META :laughing: 
It's why each time I say this is a **draft** or an **exploration**. It means we can create proposals, tweak, explore locally what we think could be a bit better and then if we are satisfied release in production. Feel free to propose things.

@magsout 
* Are you concerned because these are staying open?
* Or because there's no concrete actions?
* Would that help you if we create dependencies on some concrete actions? 

For example on #501 aka exploration on how to improve the linking system of hashtags. There is a good opportunity for a simple fix, which is just a link. Let's do that. Then we can continue the discussion about how do we make it better by either closing #501 with the simple fix or creating a new issue for the simple fix with a dependency on #501.

**Comment by miketaylr** [comment]
This issue is at least a good kick in the pants for me. :P A meta kick in the pants, perhaps.

**Comment by magsout** [comment]
@miketaylr 
>This issue is at least a good kick in the pants for me. :P A meta kick in the pants, perhaps.

I blame anyone, and this issue is not absolutely critical.

I spend less time webcompat because I have a lot of work. So I may be a little lost by coming in progress. That's why I opened this isssue to the point.

I like to work with you guys, I learnt a lot of thing with you. I want to continue to contribute to this awesome project


@karlcow 

>Are you concerned because these are staying open?
    Or because there's no concrete actions?
    Would that help you if we create dependencies on some concrete actions?

Yes for 3? 

>For example on #501 aka exploration on how to improve the linking system of hashtags. There is a good opportunity for a simple fix, which is just a link. Let's do that. Then we can continue the discussion about how do we make it better by either closing #501 with the simple fix or creating a new issue for the simple fix with a dependency on #501. 

And yes maybe. Simple fix and iterate more and more to have the best UX ever

**Comment by calexity** [comment]
@magsout thanks for getting us back on track ;) @karlcow and I had a talk prioritizing some stuff about a month ago. I put that list in my Trello board 
https://trello.com/b/YkYU1qKf/webcompat-design

I updated it with some of things you linked above. Back in December, we wrote out a bunch of milestones
https://github.com/webcompat/webcompat.com/milestones

but some more pressing things have arisen since then. I think we could use a better way to prioritize work for contributors (namely you and me and anyone else who volunteers to design or code), separate from the issues that get reported about the site.  

I think it's important that we have 
- a clear priority that everyone can see 
- a way to see who is working on what

Any ideas for a good tool/system?
